LiteCore/Query/N1QL_Parser/n1ql.leg

```diff
@@ -282,8 +282,16 @@ collateSuffix =
         )+ ')' _
     )                                   { $$ = co; }
 
+collationLang =
+    <"NOUNICODE"i>                      { $$ = string(yytext); }
+  | (
+        "UNICODE"i                      { $$ = string("UNICODE"); }
+        <(':' [a-zA-Z] [a-zA-Z0-9_]*)?> { $$ = $$.as<string>() + string(yytext); }
```

Comment:
I'm getting confused about '_' as it could also mean whitespace. Do I understand correctly that something like this is allowed, "en_US".

Reply:
yes, it is allowed as I understand. It's not?